File: 7KYU/Simple Fun #74 Growing Plant/solution.cs
namespace myjinxin
{
    using System;
    
    public class Kata
    {
        public int GrowingPlant(int upSpeed, int downSpeed, int desiredHeight)
        {
	          var height = 0;
	          int dayCount = 0;
	          while(desiredHeight > height)
	          {
					dayCount++;
		            height += upSpeed;
		            if(height >= desiredHeight)
			            return dayCount;
		            else
			            height -= downSpeed;
	          }
	          return dayCount;
        }
    }
}

File: 7KYU/Simple Fun #74 Growing Plant/solution.go
package kata

func GrowingPlant(upSpeed, downSpeed, desiredHeight int) int {
	height := 0
	dayCount := 0

	for {
		dayCount = dayCount + 1
		height = height + upSpeed

		if height >= desiredHeight {
			return dayCount
		} else {
			height = height - downSpeed
		}
	}
}
